src/frontend/src/optimizer/mod.rs

```diff
@@ -514,6 +514,16 @@ impl PlanRoot {
             ))?;
         }
 
+        if ctx.session_ctx().config().streaming_unaligned_join() {
```

Comment:
What about changing the name `streaming_unaligned_join` -> `streaming_enable_unaligned_join`?